cli/sql.go

```diff
@@ -271,6 +272,15 @@ func runTerm(cmd *cobra.Command, args []string) error {
 	if len(cliContext.execStmts) > 0 {
 		// Single-line sql; run as simple as possible, without noise on stdout.
 		return runStatements(conn, cliContext.execStmts)
+	} else if !isatty.IsTerminal(os.Stdin.Fd()) {
+		// Non-tty stdin (eg: commands piped into cockroach): run quietly.
+		// TODO(marc): we may have to read through until ';' is encountered, but
```

Comment:
What do you mean by this comment? Just reading a statement at a time instead of reading the whole thing at once?